controls/repository.sp

```diff
@@ -665,3 +681,279 @@ control "repository_plugin_squash_merge_allowed" {
       name_with_owner
   EOT
 }
+
+// Other checks
+
+control "repository_steampipe_cli_fdw_sdk_docs_description_is_set" {
+  title = "Steampipe CLI, FDW, SDK, Docs repositories have a standard description"
+  sql = <<-EOT
+    select
+      url as resource,
+      case
+        when description is not null then 'ok'
+        else 'alarm'
+      end as status,
+      name_with_owner || case
+        when description != '' then ': ' || description
+        else ' description not set'
+      || '.' end as reason,
+      name_with_owner
+    from
+      github_search_repository
+    where
+      query ='repo:turbot/steampipe repo:turbot/steampipe-plugin-sdk repo:turbot/steampipe-docs repo:turbot/steampipe-postgres-fdw is:public archived:false'
```

Comment:
Can you please add a local in `vars.sp` to store this query? This makes it easy to update in one place